app/email/email_templates.json

```diff
@@ -1,74 +1,78 @@
+  },
+  "accept_closed_review_request": {
+    "subject": "Closed review request accepted",
+    "content": "<p>Hi $preferred_username,</p> <p>$app_user ($role) has accepted the request for closed review of the following ATBD version: <a href=\"$atbd_version_link\">$atbd_title, $atbd_version</a>.</p><p>Sincerely,</p><p>The APT Team</p>"
+  },
+  "all_reviews_done": {
+    "subject": "All reviews are finished",
+    "content": "<p>Hi $preferred_username,</p> <p>All reviewers of the following ATBD version: <a href=\"$atbd_version_link\">$atbd_title, $atbd_version</a> have marked their review as DONE. It is now ready to be considered for the open review stage.</p><p>Sincerely,</p><p>The APT Team</p>"
+  },
+  "open_review": {
+    "subject": "ATBD version transitioned to open review",
+    "content": "<p>Hi $preferred_username,</p> <p>$app_user ($role) has transitioned the following ATBD version: <a href=\"$atbd_version_link\">$atbd_title, $atbd_version</a> to the open review stage.</p><p>Sincerely,</p><p>The APT Team</p>"
+  },
+  "request_publication": {
+    "subject": "Publication requested",
+    "content": "<p>Hi $preferred_username,</p> <p>$app_user has requested publication of the following ATBD version: <a href=\"$atbd_version_link\">$atbd_title, $atbd_version</a>.</p><p>Sincerely,</p><p>The APT Team</p>"
+  },
+  "cancel_publication_request": {
+    "subject": "Publication request cancelled",
+    "content": "<p>Hi $preferred_username,</p> <p>$app_user has removed their request for publication of the following ATBD version: <a href=\"$atbd_version_link\">$atbd_title, $atbd_version</a>.</p><p>Sincerely,</p><p>The APT Team</p>"
+  },
+  "publish": {
+    "subject": "Atbd version published",
+    "content": "<p>Hi $preferred_username,</p> <p>$app_user ($role) has published the following ATBD version: <a href=\"$atbd_version_link\">$atbd_title, $atbd_version</a>!</p><p>Sincerely,</p><p>The APT Team</p>"
+  },
+  "bump_minor_version": {
+    "subject": "Atbd minor version number bumped",
+    "content": "<p>Hi $preferred_username,</p> <p>$app_user ($role) has bumped the minor vesion number of the following ATBD version: <a href=\"$atbd_version_link\">$atbd_title, $atbd_version</a>!</p><p>Sincerely,</p><p>The APT Team</p>"
+  },
+  "thread_closed": {
+    "subject": "Thread closed",
+    "content": "<p>Hi $preferred_username,</p> <p>$app_user ($role) has closed a thread in the following ATBD version: <a href=\"$atbd_version_link\">$atbd_title, $atbd_version</a>, section: $section. </p><p>Sincerely,</p><p>The APT Team</p>"
+  },
+  "thread_reopened": {
+    "subject": "Thread reopened",
+    "content": "<p>Hi $preferred_username,</p> <p>$app_user ($role) has reopened a thread in the following ATBD version: <a href=\"$atbd_version_link\">$atbd_title, $atbd_version</a>, section: $section. </p><p>Sincerely,</p><p>The APT Team</p>"
+  }
```

Comment:
@wrynearson Let me know if this needs to be changed.

Reply:
@thenav56 yes, that would be great. Please use the new lines 26 and 27 [here](https://docs.google.com/spreadsheets/d/1EzsPAZ0dJNjreuEeCh3QlA3DIoFytadUxaydoH8xSds/edit#gid=260528337&range=A26)

Reply:
Done
https://github.com/NASA-IMPACT/nasa-apt/blob/735edcc1af318b5f1fbbbc853f4c110417a9e925/app/email/email_templates.json#L70-L77